ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -31,15 +37,30 @@ void RSkComponentImage::OnPaint(
     std::string path = "./" + source.uri.substr(7);
 
     Rect frame = getAbsoluteFrame();
-    SkRect rect = SkRect::MakeXYWH(frame.origin.x, frame.origin.y, frame.size.width, frame.size.height);
+    SkRect frameRect = SkRect::MakeXYWH(frame.origin.x, frame.origin.y, frame.size.width, frame.size.height);
     auto const &imageBorderMetrics=imageProps.resolveBorderMetrics(component.layoutMetrics);
 	
     /* Draw order 1. Background 2. Image 3. Border*/
     drawBackground(canvas,frame,imageBorderMetrics,imageProps.backgroundColor,imageProps.opacity);
     RNS_PROFILE_START(drawImage)
     sk_sp<SkImage> imageData=getImageData(path.c_str());
     if(imageData) {
-      canvas->drawImageRect(imageData, rect, nullptr);
+      canvas->save();
+
+      SkRect targetRect = computeTargetRect({imageData->width(),imageData->height()},frameRect,imageProps.resizeMode);
+      /* clipping logic to be applied if computed Frame is greater than the target.*/
+      if(( frameRect.width() < targetRect.width()) || ( frameRect.height() < targetRect.height())) {
+        canvas->clipRect(frameRect,SkClipOp::kIntersect);
+      }
+      SkPaint paint;
+      if(imageProps.resizeMode == ImageResizeMode::Repeat){
+        sk_sp<SkImageFilter> imageFilter(SkImageFilters::Tile(targetRect,frameRect ,nullptr));
+        paint.setImageFilter(std::move(imageFilter));
+      }
+      /*TO DO: Handle filter quality based of configuration. Setting Low Filter Quality as default for now*/
+      paint.setFilterQuality(kLow_SkFilterQuality);
```

Comment:
Can we have "kLow_SkFilterQuality" in a macro with necessary comments ?
This will help to change the configuration in future if required ?